dubbo-metrics/dubbo-metrics-config-center/pom.xml

```diff
@@ -27,7 +27,7 @@
   <dependencies>
     <dependency>
       <groupId>org.apache.dubbo</groupId>
-      <artifactId>dubbo-metrics-api</artifactId>
+      <artifactId>dubbo-metrics-core</artifactId>
```

Comment:
Can we consider relying on the API layer

Reply:
I think the current primary goal is to strip the dubbo project of its dependence on metrics and only retain its dependence on the api, so only a very small amount of content necessary for the dubbo project will be retained in the api. 
In fact, I renamed the 'dubbo-metrics-api' package to 'dubbo-metrics-core' because it contains many classes that are not APIs, such as 'CombMetricsCollector'. Then a new 'dubbo-metrics-api' was added for dubbo project integration.

Reply:
Or should I keep the original 'dubbo-metrics-api' (even though it contains a lot of non-API classes) and name the new 'dubbo-metrics-api' something like 'dubbo-metrics-client'? Looking forward to your suggestions.

Reply:
We can move towards abstraction, so I suggest relying on APIs

Reply:
Well, it's actually just a matter of package naming. I committed the changes and restored the 'dubbo-metrics-api' package name

Reply:
Well, it's actually just a matter of package naming. I committed the changes. Please confirm whether it is consistent with your suggestion